src/components/Text/Text.module.css

```diff
@@ -20,9 +20,12 @@
 .text--xs {
   @mixin eds-theme-typography-body-xs;
 }
-.text--caption {
+.text--caption, .text--caption-md {
```

Comment:
`.text-caption` is preserved for compatibility at the moment. When doing a deprecation cleanup sweep, it can be removed.